python/tutorials/10-block-scaled-matmul.py

```diff
@@ -358,12 +645,20 @@ def show_profile(profile_name):
 
         torch.manual_seed(42)
 
-        validate_block_scaled(8192, 8192, 8192, block_scale_type=args.format)
+        if is_cuda():
+            validate_block_scaled(8192, 8192, 8192, block_scale_type=args.format)
+        else:
```

Comment:
`elif is_hip_cdna4()` to make other potential out-of-tree backend's life easier.